src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs

```diff
@@ -1830,11 +1830,12 @@ private string GenerateMembersElement(XmlMembersMapping xmlMembersMapping)
 
                 for (int i = 0; i < mapping.Members.Length; i++)
                 {
+                    var iStr = i.ToString(CultureInfo.InvariantCulture);
```

Comment:
Very small nit, but the name here makes me think of the Hungarian Notation of an index that's about a string.